spec/decanter/decanter_core_spec.rb

```diff
@@ -231,6 +231,20 @@ def self.name
         end
       )
 
+      Object.const_set('KeyValueSplitterParser',
```

Comment:
Instead of pulling from the library like before, create the parser as a one-off to use in this test.